wp/sections/usage_scenarios.tex

```diff
@@ -1,6 +1,7 @@
 \subsection{Aibolit Index}
 
-In addition to the recommendation functionality, we designed a feature to measure the overall quality of developer's source code, \textit{Aibolit Index}.
+In addition to the recommendation functionality, we designed an \textit{"Aibolit Index"}. 
```

Comment:
I actually don't think you should split this sentence.

I'd just rewrite a bit: 
In addition to the recommendation functionality, we prppose \textit{Aibolit Index}, a score that measures the overall quality of developer's source code.

Reply:
I'm okay even with the previous version of the sentence, except this part `of developer's source code`. I suggest this refactoring, as I understand the professor's comments.

Reply:
Ah then maybe:

"""
In addition to the recommendation functionality, we proppose \textit{Aibolit Index}, a score that measures the overall quality of a given software project.
"""

I think "a given" sounds better than "some" in this context.

Reply:
I like it